packages/html-reporter/src/testResultView.css

```diff
@@ -40,7 +41,7 @@
 }
 
 .test-result-error-message {
-  white-space: pre;
+  white-space: pre-wrap;
```

Comment:
I don't think you can do this due to error pointer:
![wrap](https://user-images.githubusercontent.com/883973/204644088-36fc156d-5c21-4a49-a318-a25e4f2e56a3.png)

Reply:
Ah, right. Bummer!
It'd be great if playwright had proper typed diagnostics for these errors. Currently, they're just strings, so we have to unfortunately dump them as code in HTML reports.